open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/ConfigFVT.java

```diff
@@ -35,12 +39,12 @@ public static void main(String args[])
         } catch (SubjectAreaFVTCheckedException e) {
             System.out.println("ERROR: " + e.getMessage() );
         } catch (UserNotAuthorizedException | InvalidParameterException | PropertyServerException e) {
-            System.out.println("ERROR: " + e.getReportedErrorMessage() + " Suggested action: " + e.getReportedUserAction());
+            log.debug("ERROR: " + e.getReportedErrorMessage() + " Suggested action: " + e.getReportedUserAction());
```

Comment:
I suggest log.error for all errors using one of the overloaded methods that include the Throwable

Reply:
This has been taken care of. Except for the throws for IO Exception